sotodlib/toast/ops/load_context.py

```diff
@@ -207,6 +228,16 @@ class LoadContext(Operator):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
+    def _open_context(self):
+        if self.context is None:
+            return Context(self.context_file)
+        else:
+            return self.context
+
+    def _close_context(self, ctx):
+        if self.context is None:
```

Comment:
Is this sufficient or should you set `self.context` to `None`?

Reply:
There are 2 traits at play here, `context` and `context_file`.  The user specifies one but not both.  If `context` is set to a pre-existing context instance, then "open" just returns that, else it instantiates a context from `context_file` and returns.  Similarly, if `context` was given, we don't try to close it.  But if we created it then we delete it.